app/waf/src/models/monitor.model.ts

```diff
@@ -0,0 +1,287 @@
+import {model, property} from '@loopback/repository';
+import {CommonEntity} from './common.model';
+
+@model()
+export class Monitor extends CommonEntity {
```

Comment:
Putting different type of monitor attributes into one model is not a reasonable way, but should work for now. 

I think we can investigate the polymorphism which is to define multiple models for the subtype of the monitor separately. And use the parent monitor type as other model's reference. 

Could you please add a TODO item to mark this task so that you can commit the code right now with currently implemented way?

Reply:
I do some test on the inheritance of model, also the usage in repository/controller. Separate each type of monitors into different model definition should be good. 

https://github.com/zongzw/learn-typescript/tree/master/src/99.verifications/lb4/inheritance-test

Reply:
I can write that as you said, but right now I will switch to lbaasv2, setup env, learn how to write irules and investigate l7policy code. We will discuss this later.

Reply:
Good and ok for that. We'll do it in the future.